docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -183,6 +183,7 @@ The following table shows a configuration option's name, type, and the default v
 |[block-user-agents](#block-user-agents)|[]string|""|
 |[block-referers](#block-referers)|[]string|""|
 |[proxy-ssl-location-only](#proxy-ssl-location-only)|bool|"false"|
+|[default-type](#default-type)|[]string|"text/html"|
```

Comment:
this setting is just a string, not an array.

Reply:
thanks，i fixed it